wolfssl/internal.h

```diff
@@ -4460,6 +4463,11 @@ struct WOLFSSL {
 #ifdef WOLFSSL_STATIC_MEMORY
     WOLFSSL_HEAP_HINT heap_hint;
 #endif
+#ifdef WOLFSSL_DTLS
```

Comment:
Do we always need this DTLS feature? Perhaps we can add a new macro for it? Shouldn't it be disabled if building with `NO_WOLFSSL_SERVER`? I want to make sure embedded DTLS customers with small flash/ram don't have issues.

Reply:
I'm not sure putting it behind a new define makes sense. Adding a `!NO_WOLFSSL_SERVER` is a good idea since this is only useful for a server.